cmd/kueuectl/cmd/create/create_localqueue.go

```diff
@@ -0,0 +1,163 @@
+	lqExample = `  # Create a local queue 
+  kueuectl create localqueue my-local-queue -c my-cluster-queue`
+)
+
+type LocalQueueOptions struct {
+	PrintFlags *genericclioptions.PrintFlags
+
+	Name             string
+	Namespace        string
+	EnforceNamespace bool
+	ClusterQueue     v1beta1.ClusterQueueReference
+
+	UserSpecifiedClusterQueue string
+
+	Client kueuev1beta1.KueueV1beta1Interface
+
+	PrintObj func(obj runtime.Object) error
+
+	genericiooptions.IOStreams
+}
+
+func NewLocalQueueOptions(streams genericiooptions.IOStreams) *LocalQueueOptions {
+	return &LocalQueueOptions{
+		PrintFlags: genericclioptions.NewPrintFlags("created").WithTypeSetter(scheme.Scheme),
+		IOStreams:  streams,
+	}
+}
+
+func NewLocalQueueCmd(f utils.Factory, streams genericiooptions.IOStreams) *cobra.Command {
+	o := NewLocalQueueOptions(streams)
+
+	cmd := &cobra.Command{
+		Use:                   "localqueue",
+		DisableFlagsInUseLine: true,
+		Aliases:               []string{"lq"},
+		Short:                 "Creates a localqueue",
+		Long:                  lqLong,
+		Example:               lqExample,
+		Args:                  cobra.MatchAll(cobra.ExactArgs(1), cobra.OnlyValidArgs),
+		Run: func(cmd *cobra.Command, args []string) {
+			cobra.CheckErr(o.Complete(f, cmd, args))
+			cobra.CheckErr(o.Validate())
+			cobra.CheckErr(o.Run())
+		},
+	}
+
+	o.PrintFlags.AddFlags(cmd)
+
+	cmd.Flags().StringVarP(&o.UserSpecifiedClusterQueue, "clusterqueue", "c", "", "The cluster queue name which will be associated with the local queue (required).")
+	_ = cmd.MarkFlagRequired("clusterqueue")
+
+	return cmd
+}
+
+// Complete completes all the required options
+func (o *LocalQueueOptions) Complete(f utils.Factory, cmd *cobra.Command, args []string) error {
+	o.Name = args[0]
+
+	var err error
+	o.Namespace, o.EnforceNamespace, err = f.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		return err
+	}
+
+	o.ClusterQueue = v1beta1.ClusterQueueReference(o.UserSpecifiedClusterQueue)
+
+	config, err := f.ToRESTConfig()
+	if err != nil {
+		return err
+	}
+
+	clientset, err := versioned.NewForConfig(config)
+	if err != nil {
+		return err
+	}
+
+	o.Client = clientset.KueueV1beta1()
+
+	printer, err := o.PrintFlags.ToPrinter()
+	if err != nil {
+		return err
+	}
+
+	o.PrintObj = func(obj runtime.Object) error {
+		return printer.PrintObj(obj, o.Out)
+	}
+
+	return nil
+}
+
+// Validate validates required fields are set to support structured generation
+func (o *LocalQueueOptions) Validate() error {
+	if len(o.Name) == 0 {
+		return fmt.Errorf("name must be specified")
+	}
+	if len(o.ClusterQueue) == 0 {
+		return fmt.Errorf("clusterqueue must be specified")
+	}
+	if len(o.Namespace) == 0 {
+		return fmt.Errorf("namespace must be specified")
+	}
+	return nil
+}
+
+// Run create localqueue
+func (o *LocalQueueOptions) Run() error {
+	lq := o.createLocalQueue()
+	var err error
+	lq, err = o.Client.LocalQueues(o.Namespace).Create(context.TODO(), lq, metav1.CreateOptions{})
```

Comment:
```suggestion
	lq, err = o.Client.LocalQueues(o.Namespace).Create(context.Background(), lq, metav1.CreateOptions{})
```

Also: CreateOptions allows you to do a dry-run in the apiserver